.github/workflows/wait_for_required_workflows.yml

```diff
@@ -27,6 +29,10 @@ jobs:
               // This is a GitHub limitation https://github.com/cloudquery/cloudquery/issues/2688
               throw new Error('Too many files changed. Skipping check. Please split your PR into multiple ones.')
             }
+
+            const can_skip = process.env.CAN_SKIP.split("\n")
+            console.log(can_skip)
+
```

Comment:
```suggestion
```

I this log message needed? I think we can remove it now as it prints a constant value

Reply:
we need the `can_skip` variable though. I'll remove the print